docs/modules/ROOT/pages/relay.adoc

```diff
@@ -122,27 +125,50 @@ In the example above, the `transfer` transaction is signed and broadcasted by th
 
 You can read more about the web3 integration https://www.npmjs.com/package/defender-relay-client#user-content-web3js[here].
 
+[[eip1559]]
+=== EIP1559 support
+
+Since not all of the networks Defender support are EIP1559 compatible, the EIP1559 transaction support is only enabled for those **networks identified as compatible** and enabled by the team.
+
+A relayer can send EIP1559 transactions in the following ways:
+
+- Sending a transaction via UI with the EIP1559Pricing policy **enabled**
+- Sending a transaction via API with both `maxFeePerGas` and `maxPriorityFeePerGas` specified
+- Sending a transaction via API with `speed` and with the EIP1559Pricing policy **enabled**
+
+Once any transaction is sent, **it will have the same type** on every stage of its lifecycle (such as replacement and repricing), so it's currently not possible to change the type if it's already been submitted.
+
+NOTE: Any attempt to send `maxFeePerGas` or `maxPriorityFeePerGas` to non-EIP1559 compatible networks will be rejected and discarded by the relayer.
+
+You can tell if a network supports EIP1559 by looking at the Relayer <<policies, policies>>. If the EIP1559Pricing policy doesn't show up, it means that we haven't added EIP1559 support for that network.
+
+NOTE: If you notice an EIP1559 compatible network that we already support but hasn't support enabled, please don't hesitate in request it on our https://forum.openzeppelin.com/c/support/defender/36[Community Forum], or at mailto:defender@openzeppelin.com[defender@openzeppelin.com]
+
 [[speed]]
 === Speed
 
-Instead of the usual `gasPrice`, the Relayer accepts a <<relay-api-reference.adoc#send-transaction,speed>> parameter, which can be `safeLow`, `average`, `fast`, or `fastest`. These values are mapped to actual gas prices when the transaction is sent or resubmitted and vary depending on the state of the network.
+Instead of the usual `gasPrice` or `maxFeePerGas`/`maxPriorityFeePerGas`, the Relayer may also accept a <<relay-api-reference.adoc#send-transaction,speed>> parameter, which can be `safeLow`, `average`, `fast`, or `fastest`. These values are mapped to actual gas prices when the transaction is sent or resubmitted and vary depending on the state of the network.
+Beware that if speed is provided, the transaction would be priced according to the <<relay-api-reference.adoc#relayer-policies, `EIP1559Pricing` relayer policy>>.
```

Comment:
Not sure I understand this sentence

Reply:
Which part do you think is not clear?
I can rephrase but I don't want to make it even harder to read

Reply:
Done, thanks for the detail Palla!